crates/rust-analyzer/src/lsp_ext.rs

```diff
@@ -202,6 +218,8 @@ impl Request for CodeActionRequest {
 pub struct CodeAction {
```

Comment:
This is missing `pub diagnostics: Option<Vec<Diagnostic>>,`. In addition it is going to conflict with `edit` in 3.16.

Reply:
We don’t use diagnostics, so that’s why it is missing. What exactly changes in edit in 3.16?

Reply:
Ignore me. I was thinking of insert/replace edits in `CompletionItem`